File: exercicios-python/1159 - SomaParesConsecutivos.py
contador = soma = 0
while(True):
    valor = int(input())
    if(valor == 0):
        break
    while contador < 5:
        if valor % 2 == 0:
            soma += valor
            contador += 1
        valor += 1
    print(f'{soma}')
    contador = soma = 0

File: exercicios-javascript/1159 - SomaParesConsecutivos.js
var input = require('fs').readFileSync('JavaScript/stdin', 'utf8');
var lines = input.split('\n')

var contador = soma = 0

while(true){
    var valor = Number(lines.shift())
    if(valor == 0){
        break
    }
    while(contador < 5){
        if(valor % 2 ==  0){
            soma += valor
            contador += 1
        }
        valor += 1
    }
    console.log(soma)
    contador = soma = 0
}
